### PR TITLE
Disable redirect

### LINKED ISSUE #1866: Verify that files are correctly downloaded in Makefile
The Makefile downloads a couple of files using wget (for example several JARs for MFP). If there is a redirect or if the requests fail, the target file is created anyway and the build does not fail.

There should be check that the return code is `200`.

### PR DESCRIPTION
**Pull request by sbrunner**
fix  #1866

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -750,7 +750,7 @@ print/WEB-INF/lib/jasperreports-fonts-5.5.0.jar:
 
 print/WEB-INF/lib/postgresql-9.3-1102.jdbc41.jar:
 	mkdir -p $(dir $@)
-	wget http://jdbc.postgresql.org/download/postgresql-9.3-1102.jdbc41.jar -O $@
+	wget --max-redirect=0 https://jdbc.postgresql.org/download/postgresql-9.3-1102.jdbc41.jar -O $@
```
Comment (ger-benjamin) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: From http to https ? it's correct ?

**Comment by ger-benjamin** [comment]
Read my comment then (correct it and then) LGTM

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Yes because the http is redirected to the https :-)

**Comment by tsauerwein** [comment]
Sorry, I missed this PR. With this change you are avoiding the redirect, but you will create an empty file and no error will be emitted. Or not?

**Comment by sbrunner** [comment]
wget steal write a file :-/ just to be against Make?

**Comment by sbrunner** [comment]
I will use curl ...

**Pull request merged**